src/index.html

```diff
@@ -1,13 +1,334 @@
+                <img 
+                  src="./images/Boss_img/gallery/5.png" 
+                  alt="Headphones & earbuds"
+                  class="place__photo"
+                >
+              </div>
+
+              <div class="place__photo-container">
+                <img 
+                  src="./images/Boss_img/gallery/6.png" 
+                  alt="Headphones & earbuds"
+                  class="place__photo"
+                >
+              </div>
+            </div>
+  
+            <div class="place__description">
+              Headphones & earbuds
+            </div>
+          </div>
+
+          <div class="place__button-container">
+            <div class="place__button-position">
+              <a href="/" class="button">
+                View all
+              </a>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="page__section about-us" id="about-us">
+        <div class="container ">
+          <div class="page__section-text-container">
+            <h2 class="page__section-title page__section-title--dif">
+              Why buy direct<br>from Bose
+            </h2>
+          
+
+            <div class="page__section-text-wrap">
+              <div class="page__section__text-bold">
+                Free 2-day shipping and returns
+                <br><br>
+                90-day risk-free trial
+                <br><br>
+                World class customer service
+                <br><br>
+                My Bose account management
+                <br><br>
+              </div>
+  
+              <div class="page__section__text-normal">
+                A great product is more than what’s in the box. It’s a promise of premium performance, world-class support, and everything you expect from a trusted brand. It’s just one of many reasons why you’ll shop with confidence on Bose.com.
+              </div>
+            </div>
+            
+          </div>
+        </div>
+
+      </section>
+
+      <section class="page__section contact-us" id="contact-us">
+        <div class="container">
+          <h2 class="page__section-title"> 
+            Contact us
+          </h2>
+          <div class="contact-us-container">
+            <form action="/" class="contact-us__form">
+              <input 
+                name="name" 
+                type="text"
+                placeholder="Name"
+                class="contact-us__field form-field"
+              >
+              <input 
+                name="email" 
+                type="email"
+                placeholder="Email"
+                class="contact-us__field form-field">
+              <textarea 
+                name="message"
+                placeholder="Message"
+                class="
+                        contact-us__field 
+                        form-field 
+                        form-field--textarea"
+              ></textarea>
+              <a href="button" class="button">Send</a>
```

Comment:
That's what I see when I click on button
![image](https://user-images.githubusercontent.com/37161315/170693344-7640bfa1-dff8-47f0-9504-a8cb16b840a3.png)